themes/vocabulary_theme/templates/page-with-toc.html

```diff
@@ -118,16 +117,16 @@ <h1>{{ this.title }}</h1>
             {% endif %}
           </div>
           <div class="row padding-bottom-normal sticky-top">
-            <h4 class="b-header">On this page</h4>
+            <h4 class="b-header" id="on-this-page-header">On this page</h4>
             <aside class="menu table-of-contents">
               <ul class="menu-list">
                 {% for item in this.body.toc %}
                   <li>
-                    <a class="link" href="#{{ item.anchor }}">{{ item.title }}</a>
+                    <a class="link" href="#{{ item.anchor }}" aria-label="{{ item.title }}">{{ item.title }}</a>
```

Comment:
if the `anchor` encloses` descriptive content, it shouldn't need an `aria-label`